mne/io/edf/edf.py

```diff
@@ -285,10 +285,8 @@ def _get_edf_info(fname, stim_channel, annot, annotmap, eog, misc, preload):
 
     with open(fname, 'rb') as fid:
         assert(fid.tell() == 0)
-        fid.seek(8)
+        fid.seek(168)  # Seek 8 + 80 bytes for Subject id + 80 bytes for rec id
 
-        fid.read(80).strip().decode()  # subject id
-        fid.read(80).strip().decode()  # recording id
```

Comment:
I agree. This is just a quick fix to be able to read data.

We need to properly handle those strings.

> El 26 may 2016, a las 14:03, Eric Larson <notifications@github.com> escribió:
> 
> In mne/io/edf/edf.py <https://github.com/mne-tools/mne-python/pull/3250#discussion_r64733553>:
> 
> >  
> > -        fid.read(80).strip().decode()  # subject id
> > -        fid.read(80).strip().decode()  # recording id
> Shouldn't we actually use these to populate entries in info? Or do we already get them from somewhere else?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/mne-tools/mne-python/pull/3250/files/973292e051f6043808ca723aeaf29afbf2c7f56a#r64733553>